modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -113,3 +120,5 @@ void register_lsp_types() {
 	GDREGISTER_CLASS(GDScriptTextDocument);
 	GDREGISTER_CLASS(GDScriptWorkspace);
 }
+
+int GDScriptLanguageServer::port_override = -1;
```

Comment:
Nit: We usually initialize statics at the top of the .cpp, not at the bottom.

Reply:
Fixed